proof/refine/ARM/CSpace_I.thy

```diff
@@ -14,8 +14,8 @@ begin
 
 context begin interpretation Arch . (*FIXME: arch_split*)
 
-lemmas capUntypedPtr_simps[simp] = capUntypedPtr_def[split_simps capability.split]
-lemmas arch_capUntypedPtr_simps[simp] = ARM_H.capUntypedPtr_def[split_simps arch_capability.split]
+lemmas capUntypedPtr_simps[simp] = capUntypedPtr_def[split_simps capability.split, simplified PPtr_def id_def]
+lemmas arch_capUntypedPtr_simps[simp] = ARM_H.capUntypedPtr_def[split_simps arch_capability.split, simplified PPtr_def id_def]
```

Comment:
I'm not convinced by these changes, but this makes these lemmas the same as on master and means that I don't need to unfold `PPtr_def` in later proofs.

Reply:
Hmm. I don’t have a particular view on this. I suppose this should be fine for the current state of Refine.

Reply:
No problem from my side. It's probably not really consistent currently anyway and if it helps, go for it.